base/actions/actions.py

```diff
@@ -636,7 +636,10 @@ async def run(
 
             risk = utils.get_risk_level(data)
 
-            patch_data = {"data": {"follow_up_optin": True}}
+            patch_data = {
+                "language": config.LANGUAGE,
+                "data": {"follow_up_optin": True},
+            }
```

Comment:
We can add a `patch_data: dict = ...` annotation here, which will tell mypy that the dictionary keys/values can be of any type (since we're not really enforcing a specific format for this dictionary)